Bottom: f9d2c210f0f1477a86724bc9aadb4ec45ab0bc94
Top:    e98a34b71c4b83e7d191fbe9ac3bd36ce0bb716f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-09 16:17:47 +0000

Refresh of make-hid_draw-an-instance-of

---

diff --git a/src/hid/batch/batch.c b/src/hid/batch/batch.c
index 27bcc71..252e2bf 100644
--- a/src/hid/batch/batch.c
+++ b/src/hid/batch/batch.c
@@ -330,7 +330,7 @@ hid_batch_init ()
 {
   memset (&batch_hid, 0, sizeof (HID));
   memset (&batch_graphics, 0, sizeof (HID_DRAW));
-  memset (&batch_graphics_ckass, 0, sizeof (HID_DRAW_CLASS));
+  memset (&batch_graphics_class, 0, sizeof (HID_DRAW_CLASS));
 
   common_nogui_init (&batch_hid);
 
diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 7efb900..fbfcc7f 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -4135,7 +4135,7 @@ hid_lesstif_init ()
   lesstif_graphics_class.fill_polygon   = lesstif_fill_polygon;
   lesstif_graphics_class.fill_rect      = lesstif_fill_rect;
 
-  lesstif_graphics.draw_pcb_polygon    = common_gui_draw_pcb_polygon;
+  lesstif_graphics_class.draw_pcb_polygon = common_gui_draw_pcb_polygon;
 
   lesstif_graphics.klass = &lesstif_graphics_class;
   common_draw_helpers_init (&lesstif_graphics);
